demonstrations/getting_started_with_hybrid_jobs.py

```diff
@@ -0,0 +1,401 @@
+# create a hybrid job with the Rigetti devices as the priority QPU. We also increase the number of
+# steps to 10.
+#
+# Using hybrid jobs for iterative algorithms is very benefit because you retain priority access to the
+# target QPU. So once your quantum tasks are created in the hybrid job, they run ahead of other tasks
+# waiting in the regular quantum task queue. This is because hybrid jobs has a separate queue from
+# standalone tasks so that only a single hybrid job can run on a QPU at a time. This means your
+# algorithm will not be interrupted by other quantum tasks, so it will run more efficiently and
+# predictably. However, hybrid jobs has a separate queue from standalone tasks so that only a single
+# hybrid job can run on a QPU at a time. So for a single quantum circuit, or a batch of circuit, it’s
+# recommended to create quantum tasks instead of hybrid jobs.
+#
+# To get QPU priority, you must ensure that the device ARN used within the function matches that
+# specified in the decorator. For convenience, you can use the helper function ``get_device_arn()`` to
+# automatically capture the device ARN declared in ``@hybrid_job``.
+#
+# In the previous example, we declared the local simulator outside the decorated function scope.
+# However, for AWS devices such as QPUs or on-demand simulators, the device must be declared within
+# the function scope. This is avoid unintentional sharing of AWS credentials.
+#
+# .. note::
+#   AWS devices must be declared within the body of the decorated function.
+#
+
+from braket.devices import Devices
+
+device_arn = Devices.Rigetti.AspenM3.value
+
+
+@hybrid_job(device=device_arn)
+def qpu_qubit_rotation_hybrid_job(num_steps=10, stepsize=0.5):
+    # AWS devices must be declared within the decorated function.
+    device = qml.device(
+        "braket.aws.qubit",
+        device_arn=device_arn,  # Make sure the device ARN matches the hybrid job device ARN
+        wires=2,
+        shots=1_000,
+    )
+
+    @qml.qnode(device)
+    def circuit(params):
+        qml.RX(params[0], wires=0)
+        qml.RY(params[1], wires=0)
+        return qml.expval(qml.PauliZ(0))
+
+    opt = qml.GradientDescentOptimizer(stepsize=stepsize)
+    params = np.array([0.5, 0.75])
+
+    for i in range(num_steps):
+        # update the circuit parameters
+        params = opt.step(circuit, params)
+        expval = circuit(params)
+
+        log_metric(metric_name="expval", iteration_number=i, value=expval)
+
+    return params.tolist()
+
+
+######################################################################
+# To get a sense of how long we will wait before the hybrid job runs, we can check the hybrid job
+# queue depth with ``AwsDevice(device_arn).queue_depth().job``. We can also check if the device is
+# currently available with ``AwsDevice(device_arn).is_available()``.
+#
+# When there are no other hybrid jobs in the queue ahead of you, and the device is available, the
+# hybrid job will start running.
+# ipy
```

Comment:
```suggestion
#
```

I'm not sure what this ipy was doing here. I delete it